ani-cli

```diff
@@ -465,44 +435,44 @@ open_episode () {
 				fi
 				;;
 		esac
-		PID=$!
-
+		PID="$!"
 		mv "${logfile}.new" "$logfile"
 	else
 		mkdir -p "$download_dir"
-		inf "Downloading episode $episode ...\n"
 		# add 0 padding to the episode name
-		episode=$(printf "%03d" "$episode")
-		{
-			#uncomment this below line if you are getting low download speeds, and comment next one after below line
-			#aria2c -x 16 -s 16 --referer="$dpage_link" "$video_url" --dir="$download_dir" -o "${anime_id}-${episode}.mp4" --download-result=hide &&
-				if aria2c --referer="$dpage_link" "$video_url" --dir="$download_dir" -o "${anime_id}-${episode}.mp4" --download-result=hide ;then
-					inf "Downloaded episode: $episode"
-				else
-					err "Download failed episode: $episode , please retry or check your internet connection"
-				fi
-		}
+		episode=$(printf '%03d' "$episode")
+		inf "Downloading episode $episode ..."
+		#uncomment this below line if you are getting low download speeds, and comment next one after below line
+		#aria2c -x 16 -s 16 --referer="$dpage_link" "$video_url" --dir="$download_dir" -o "${anime_id}-${episode}.mp4" --download-result=hide &&
+		if aria2c --referer="$dpage_link" "$video_url" --dir="$download_dir" -o "${anime_id}-${episode}.mp4" --download-result=hide; then
+			inf "Downloaded episode: $episode"
+		else
+			err "Download failed episode: $episode , please retry or check your internet connection"
+		fi
 	fi
 }
 
+
 ############
-# Start Up #
+# START UP #
 ############
 
-# to clear the colors when exited using SIGINT
-trap 'printf "\033[0m"; exit 1' INT HUP
+# default options
+player_fn='mpv' #video player needs to be able to play urls
+is_download="0"
+half_ep="0"
+quality='best'
+scrape='query'
+download_dir='.'
+choice=''
```

Comment:
I see that you have done a lot of
```
variable=''
```
The cleaner way would be
```
variable=
```
just leave a blank

Reply:
Yes, it does appear cleaner, I'll change it